assume/common/utils.py

```diff
@@ -291,6 +289,8 @@ def aggregate_step_amount(orderbook: Orderbook, begin=None, end=None, groupby=[]
 
     If called without groupby, this returns the aggregated orderbook timeseries
     """
+    if groupby is None:
+        groupby = []
```

Comment:
What is the benefit of this?

Reply:
before the groupby was  mutable argument, which is not desired. so I replaced it but this is the work around